index.php

```diff
@@ -1,181 +1,13 @@
 <?php
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+include_once 'vendor/autoload.php';
 include_once 'language.php';
 include_once 'api/get_rbg_streams.php';
```

Comment:
nit: at some point we should setup PSR-4 autoloading for all the other files as well (but that is for another day)